lib/utils.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+exports.makeContinuation = (predicate) => {
+
+    return (callback) => {
+
+        process.nextTick(() => {
```

Comment:
Since you're targeting new versions of Node, you can do `process.nextTick(callback, null, predicate());`